lib/yajsonrpc/__init__.py

```diff
@@ -1,18 +1,6 @@
-# Copyright (C) 2014-2019 Red Hat Inc.
-# Copyright (C) 2014 Saggi Mizrahi, Red Hat Inc.
```

Comment:
It would be nice to keep Saggi copyright (as a second SPDX-FileCopyrightText)

Reply:
Right, thanks! I have still missed one :facepalm: